examples/simple/game.py

```diff
@@ -0,0 +1,34 @@
+from comprl.server.data.interfaces import GameEndState, GameResult
+from comprl.server.interfaces import IGame, IPlayer
+
+
+class ExampleGame(IGame):
+    """example for a game"""
+
+    def __init__(self, players: list[IPlayer]) -> None:
+        super().__init__(players=players)
+        self.env = 0
+
+    def _update_environment(self):
+        for action in self.current_actions:
+            self.env += sum(action)
+
+    def _validate_action(self, action):
+        return isinstance(action, int)
+
+    def _is_finished(self) -> bool:
+        return self.env > 10
+
+    def _observation(self, index: int = 0) -> list[float]:
+        return [float(self.env)]
+
+    def _player_stats(self, index) -> int:
+        return 0
+
+    def _player_won(self, index) -> bool:
+        if index == 0:
+            return True
+        return False
+
+    def get_result(self) -> GameResult:
+        return GameResult(self.id, -1, -1, 0, 0, None, GameEndState.DRAW, True, True)
```

Comment:
It would be better to use the actual `user_id`s here, because it simulates a real game better.

If you would try to do this here, it would cause a problem, because a player doesn't know its `user_id` and the game doesn't know the player manager, who has the `user_id`. This means there is currently no way to obtain the `user_id` in this class, so some restructuring has to be done here.

Reply:
Yeah, I thought about storing the user_id in the player obejcts so we could reduce expensive IO calls